### PR TITLE
[cereal] Update to 1.3.1

### PR DESCRIPTION
**Pull request by mapret**
- #### What does your PR fix?  
  See [cereal Changelog 1.3.1](https://github.com/USCiLab/cereal/releases/tag/v1.3.1)

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

File: ports/cereal/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "cereal",
-  "version-string": "1.3.0",
-  "port-version": 1,
+  "version-string": "1.3.1",
```
Comment (JonLiu1993) [inline ports/cereal/vcpkg.json:3]:
```suggestion
  "version": "1.3.1",
```

Comment (mapret) [reply ports/cereal/vcpkg.json]: Done

File: ports/cereal/portfile.cmake [hunk 2]
```diff
@@ -16,10 +16,11 @@ vcpkg_configure_cmake(
 )
 
 vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/cmake/cereal)
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/cereal)
 
 # Clean
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib)
```
Comment (JonLiu1993) [inline ports/cereal/portfile.cmake:23]: @mapret ,Could you please clarify why remove lib directory?

**Comment by JonLiu1993** [comment]
Looks openmvg build failed ,
This is error log, please take a look:
````
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:128 (message):
    Command failed: ninja -v
    Working Directory: D:/buildtrees/openmvg/x86-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      D:\buildtrees\openmvg\config-x86-windows-out.log
````

````
[1/2] "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/openmvg/x86-windows-rel" "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvg/src/1047522df9-4334301758.clean/src" "-DOpenMVG_USE_OPENCV=OFF" "-DOpenMVG_USE_OCVSIFT=OFF" "-DOpenMVG_BUILD_SOFTWARES=ON" "-DOpenMVG_BUILD_GUI_SOFTWARES=ON" "-DOpenMVG_USE_OPENMP=OFF" "-DOpenMVG_BUILD_SHARED=OFF" "-DOpenMVG_BUILD_TESTS=OFF" "-DOpenMVG_BUILD_DOC=OFF" "-DOpenMVG_BUILD_EXAMPLES=OFF" "-DOpenMVG_BUILD_OPENGL_EXAMPLES=OFF" "-DOpenMVG_BUILD_COVERAGE=OFF" "-DOpenMVG_USE_INTERNAL_CLP=OFF" "-DOpenMVG_USE_INTERNAL_COINUTILS=OFF" "-DOpenMVG_USE_INTERNAL_OSI=OFF" "-DOpenMVG_USE_INTERNAL_EIGEN=OFF" "-DOpenMVG_USE_INTERNAL_CEREAL=OFF" "-DOpenMVG_USE_INTERNAL_CERES=OFF" "-DOpenMVG_USE_INTERNAL_FLANN=OFF" "-DOpenMVG_USE_INTERNAL_LEMON=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvg_x86-windows"
FAILED: ../CMakeCache.txt 
"D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/openmvg/x86-windows-rel" "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvg/src/1047522df9-4334301758.clean/src" "-DOpenMVG_USE_OPENCV=OFF" "-DOpenMVG_USE_OCVSIFT=OFF" "-DOpenMVG_BUILD_SOFTWARES=ON" "-DOpenMVG_BUILD_GUI_SOFTWARES=ON" "-DOpenMVG_USE_OPENMP=OFF" "-DOpenMVG_BUILD_SHARED=OFF" "-DOpenMVG_BUILD_TESTS=OFF" "-DOpenMVG_BUILD_DOC=OFF" "-DOpenMVG_BUILD_EXAMPLES=OFF" "-DOpenMVG_BUILD_OPENGL_EXAMPLES=OFF" "-DOpenMVG_BUILD_COVERAGE=OFF" "-DOpenMVG_USE_INTERNAL_CLP=OFF" "-DOpenMVG_USE_INTERNAL_COINUTILS=OFF" "-DOpenMVG_USE_INTERNAL_OSI=OFF" "-DOpenMVG_USE_INTERNAL_EIGEN=OFF" "-DOpenMVG_USE_INTERNAL_CEREAL=OFF" "-DOpenMVG_USE_INTERNAL_CERES=OFF" "-DOpenMVG_USE_INTERNAL_FLANN=OFF" "-DOpenMVG_USE_INTERNAL_LEMON=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvg_x86-windows"
-- The C compiler identification is MSVC 19.29.30139.0
-- The CXX compiler identification is MSVC 19.29.30139.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- target changed from "" to "auto"
-- Detected CPU: generic
-- Checking for C++11 compiler
-- Checking for C++11 compiler - available
-- Found JPEG: optimized;D:/installed/x86-windows/lib/jpeg.lib;debug;D:/installed/x86-windows/debug/lib/jpeg.lib (found version "62") 
-- Found ZLIB: optimized;D:/installed/x86-windows/lib/zlib.lib;debug;D:/installed/x86-windows/debug/lib/zlibd.lib (found version "1.2.11") 
-- Found PNG: optimized;D:/installed/x86-windows/lib/libpng16.lib;debug;D:/installed/x86-windows/debug/lib/libpng16d.lib (found version "1.6.37") 
-- Found TIFF: optimized;D:/installed/x86-windows/lib/tiff.lib;debug;D:/installed/x86-windows/debug/lib/tiffd.lib (found version "4.3.0")  
CMake Error at CMakeLists.txt:255 (get_target_property):
  get_target_property() called with non-existent target "cereal".
````

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**